code/datums/traits/negative.dm

```diff
@@ -174,9 +182,10 @@
 
 	if(!heirloom_type)
 		heirloom_type = pick(
-		/obj/item/toy/cards/deck,
-		/obj/item/lighter,
-		/obj/item/dice/d20)
+		/obj/item/reagent_containers/food/drinks/urn,
```

Comment:
this means the datum is tied to the urn, not the ashes- drinking the ashes has no negative effect